templates/CRM/Admin/Form/Preferences/Display.tpl

```diff
@@ -228,6 +239,18 @@
           $('#contact_edit_preferences').val(params.toString());
         }
 
+        //show/hide activity types based on checkbox value
+        $('#filter_activity_type_notification').hide();
+        $('#filter_activity_type_notification-desc').hide();
+        if ($('#activity_assignee_notification').is(":checked")) {
+          $('#filter_activity_type_notification').show();
+          $('#filter_activity_type_notification-desc').show();
+        }
+        $('#activity_assignee_notification').change(function() {
+          $('#filter_activity_type_notification').toggle();
+          $('#filter_activity_type_notification-desc').toggle();
+        });
+
```

Comment:
This javascript is overly complicated and prone to failure, since the `.toggle()` function is being used without any arguments passed in, which means it could get out-of-sync with the checkbox. I'm also not a fan of how it hides the element then immediately shows it again, causing a potential annoying flash on page load.

Why is it necessary to target two elements `filter_activity_type_notification` and `filter_activity_type_notification-desc` instead of using a wrapper around both?